CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Instructions
+
+- Fork the CadQuery repository, clone your fork and create a new branch to start working on your changes
+- Create a conda development environment with something like: 
+`conda env create -n cq-cam-dev -f environment.yml`
```

Comment:
An alternative approach here is to just setup the conda with cadquery instructions. I might add that later in, no need to include it now.